src/components/Picker/Picker.tsx

```diff
@@ -0,0 +1,156 @@
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { BasePicker, IBasePickerProps, IPickerItemProps } from 'office-ui-fabric-react/lib/Pickers';
+import Block, { TextSize } from '../Block';
+import { FixedGridRow, FixedGridColumn } from '../FixedGrid';
+import SuggestionItemContent, { SuggestionItem } from '../SuggestionsList/SuggestionsListItemContent';
+import RemoveIcon from '../Icon/icons/Cancel';
+import { getClassNames } from './Picker.styles';
+import './Picker.css'; // Required for the Callout content as there aren't JS styles to override
+
+class SuggestionListBasePicker extends BasePicker<SuggestionItem, IBasePickerProps<SuggestionItem>> {}
+
+export interface BasePickerProps<T> extends BaseComponentProps {
+  /**
+   * This is where you provide the items to be rendered in the autocomplete dropdown.
+   * The current filter text and list of already-selected items will be provided.
+   */
+  onResolveSuggestions: IBasePickerProps<T>['onResolveSuggestions'];
+
+  /**
+   * Called when the list of selected items changes.
+   */
+  onChange: IBasePickerProps<T>['onChange'];
+
+  /**
+   * The list of currently-selected items.
+   */
+  selectedItems: IBasePickerProps<T>['selectedItems'];
+
+  /**
+   * Aria label for the focusable text input.
+   */
+  inputAriaLabel: string;
+
+  /**
+   * Aria label for the close button on selected items.
+   */
+  removeItemAriaLabel: string;
+
+  /**
+   * Visible header text in the autocomplete dropdown.
+   */
+  suggestionsHeaderText: string;
+
+  /**
+   * Message to display when no results are found.
+   */
+  noResultsFoundText: string;
+
+  /**
+   * Disables the picker UI.
+   */
+  disabled?: boolean;
+
+  /**
+   * If provided, limits the number of items which can be selected.
+   */
+  itemLimit?: number;
+
+  /**
+   * Callback event when the UI element is focused.
+   */
+  onFocus?: IBasePickerProps<T>['onFocus'];
+
+  /**
+   * Callback event when the UI element loses focus.
+   */
+  onBlur?: IBasePickerProps<T>['onBlur'];
+}
+
+export type PickerProps = BasePickerProps<SuggestionItem>;
+
+/**
+ * Pickers are used to pick one or more entities (e.g. people, groups, topics) from a list with typeahead capabilities.
+ */
+export default class Picker extends React.Component<PickerProps> {
+  public render() {
+    const {
+      className,
+      disabled,
+      itemLimit,
+      onFocus,
+      onBlur,
+      onResolveSuggestions,
+      onChange,
+      selectedItems,
+      inputAriaLabel,
+      suggestionsHeaderText,
+      noResultsFoundText,
+    } = this.props;
+
+    return (
+      <SuggestionListBasePicker
+        className={join(['y-picker', className])}
+        onResolveSuggestions={onResolveSuggestions}
+        pickerSuggestionsProps={{
+          suggestionsHeaderText,
+          noResultsFoundText,
+        }}
+        itemLimit={itemLimit}
+        disabled={disabled}
+        inputProps={{
+          onBlur: onBlur,
+          onFocus: onFocus,
```

Comment:
object shorthand?

Reply:
I guess we dont lint for it in YamUI?

Reply:
Yeah, difficult to keep the two sets of configs in sync. Will fix.